website/content/docs/commands/tls/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+page_title: 'Commands: TLS'
+---
```

Comment:
This page should just be an overview with links to subcommands. See https://www.nomadproject.io/docs/commands/alloc for an example. We don't need pointers to the sidebar or the `--help` text or examples. All of that belongs in the individual subcommand docs.